identity/test/pool.go

```diff
@@ -134,7 +132,6 @@ func TestPool(ctx context.Context, conf *config.Config, p interface {
 					assert.Empty(t, actual.RecoveryAddresses)
 					assert.Empty(t, actual.VerifiableAddresses)
 
-					require.Len(t, actual.InternalCredentials, 2)
```

Comment:
Reason for removal?

Reply:
Software that imports kratos as a package might choose to hydrate the credentials in a different way, bypassing `InternalCredentials`. This tests implementational details, which we shouldn't do IMO.

LMKWYT.